app/services/notify_service.rb

```diff
@@ -126,7 +126,7 @@ def drop?
     private
 
     def blocked_mention?
-      FeedManager.instance.filter?(:mentions, @notification.target_status, @recipient)
+      !!FeedManager.instance.filter?(:mentions, @notification.target_status, @recipient)
```

Comment:
There is an argument to be made that `filter` and `filter?` should become two seperate functions, one just testing the truthiness of the other, since else it results in stuff like this, but i'll leave it up to y'all to decide on that.